### ISSUE TITLE
ExpandableSection - examples missing aria attributes

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7280]
**Describe the enhancement or change**
- The toggle needs the `aria-controls` attribute pointing to an ID of the expandable section content
- The expandable section content that can be collapsed should have a role of "region" and an aria-labelledby pointing to the toggle text, to match React.

**Any other information?**

**Comment by Mash707** [comment]
@thatblindgeye could you assign this to me.
